algorithms/CPlusPlus/Codes/huffmanCode/huffman.h

```diff
@@ -0,0 +1,33 @@
+#include <string>
```

Comment:
It's recommend to add a [header guard](https://en.wikipedia.org/wiki/Include_guard) to your header file.

Reply:
this is also the reason why `cpplint` check is failing.